package.json

```diff
@@ -31,6 +31,9 @@
     "debowerify": "^1.3.1",
     "del": "^2.2.0",
     "eslint-plugin-react": "^3.6.3",
+    "connect-history-api-fallback": "1.1.0",
+    "firebase": "2.3.1",
+    "history": "^1.17.0",
```

Comment:
It seems like these additions aren't related to the PR ... to reduce confusion later, you might want to remove these from here if they're not needed.